src/Componants/Heading.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const Heading = () => {
+  return (
+    <div className="main-head">
+      <img
+        src="https://image.flaticon.com/icons/svg/139/139899.svg"
+        alt="Hotal icon"
+        width="3%"
```

Comment:
We generally try to avoid inline styling. What about move this 3% to a CSS class instead?

How did you come up with this number? Why 3%?